zip-draft-XXXX.rst

```diff
@@ -0,0 +1,289 @@
+   complies with applicable laws and regulations.
+
+4. **Technical Implementation**: The technical specifics of how the lockbox
+   will be integrated into the Zcash protocol, including any necessary changes
+   to the codebase and the precise methods for ensuring security and
+   transparency, are not covered in this ZIP. These details will be provided in
+   subsequent proposals and technical documentation.
+
+5. **Taxation Strategy**: Although the potential for improved tax efficiency is
+   mentioned, this ZIP does not provide a detailed strategy for managing tax
+   liabilities. Future ZIPs will explore the implications of tax compliance and
+   the best methods for handling taxable events related to the disbursement of
+   funds.
+
+6. **Impact Assessment**: The long-term impact of reallocating a portion of the
+   block reward to the lockbox on the Zcash ecosystem, including its effect on
+   miners, developers, and the broader community, is not analyzed in this ZIP.
+   Subsequent proposals will need to evaluate the outcomes and make necessary
+   adjustments based on real-world feedback and data.
+
+By focusing on the establishment of the lockbox, this ZIP aims to lay the
+groundwork for a more decentralized and efficient funding mechanism. The
+deferred issues will be crucial in shaping the final implementation and
+ensuring the proposed system's success and sustainability.
+
+Specification
+=============
+
+This section describes the consensus rules and protocol changes required to
+implement the proposed in-protocol lockbox for deferred development fund
+allocation. The specification outlines the creation, management, and
+integration of the lockbox within the Zcash protocol.
+
+Deferred Development Fund Chain Value Pool Balance
+--------------------------------------------------
+
+A new development fund value pool balance, denominated in zatoshis, will be
+created. This balance will be initialized to zero at the NU6 activation block.
+
+Coinbase Transaction Adjustments
+--------------------------------
+
+For coinbase transactions, a specified proportion of the block subsidy will be
+added to the deferred development fund chain value pool balance and subtracted
+from the transparent transaction value pool. This subtraction will occur after
+block reward value is logically added to the transparent value pool. The exact
+proportions (numerator/denominator) of the block subsidy allocated to the
+deferred development fund must be defined to ensure the correct allocation at
+every block.
+
+Proportion Specification
+------------------------
+
+The proportions of the block subsidy allocated to the deferred development fund
+and any existing funding streams must add up to the intended total at every
+block. Before NU6 activation, the actual percentage allocated to the deferred
+development fund will be zero. After NU6 activation, |percentage| of the block
+reward will be allocated to the deferred development fund.
```

Comment:
This was fine as a comment in the HackMD it was taken from, but is confusing here (as this ZIP draft does not propose for any funding streams to be concurrent with the lockbox). Something like it may end up in the ZIP that defines how the core lockbox / issuance pool mechanism works, but for today's deadline, remove all but the last sentence:
```suggestion
After NU6 activation, |percentage| of the block reward will be allocated to the
deferred development fund.
```